src/css/custom.css

```diff
@@ -9,6 +9,11 @@
   --ifm-color-primary: #3071a9;
   --ifm-code-font-size: 95%;
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.1);
+  --ifm-footer-background-color:rgba(115, 150, 233, 0.61);
+  /* --ifm-footer-link-color:rgba(0, 0, 0, 1); */
+  /* --ifm-footer-link-font-family:'PingFang SC-Light';
+  --ifm-footer-link-font-size:28pt;
+  --ifm-footer-link-font-weight:light; */
```

Comment:
I think you have to make some note comments to explain why there are several commented code line.

Reply:
Sorry, my personal coding habits are not standardized enough, and I forgot to remove these commented out invalid codes. I will pay attention to these details later.